### ISSUE TITLE
Image missing a text alternative (thumbnails on search results page)

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5342]
Sites identified: Archivo, IJCCR
Page(s) identified: 

- Search results page (thumbnail image)

WCAG compliance level: A

Description:

All non-decorative images must have a text alternative (also known as "alt text").

Text alternatives are used to describe images to people who are unable to see them. This includes people who use screen readers to access the web.

The text alternative should describe the image within the context of the page — letting visitors know what the image means and why it was included.

For more details, see [SiteImprove example](https://my2.siteimprove.com/Inspector/1166400/A11Y/Page?pageId=42870396830&impmd=DB21396D67119C3E30B314CCF5B34C39&conf=a+aa+aria+si&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r2/failed/c013a1RBSGk=;ODA4OGM2M2I=)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBb2xnIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--ad283e63c30789a148dd6c809ead4e147ada6e50/image.png)

Note: This was not identified on MCO search results pages by the scanner, but it's likely the same issue occurs there.

# Done Looks Like
- Alt text attribute exists for thumbnail images in catalog view/Blacklight and is an empty string to indicate a decorative image

**Comment by joncameron** [comment]
Falls under decorative? https://www.w3.org/WAI/tutorials/images/decorative/
The screenshot may not be listed under meaningful text to the user.
In the future it could be possible to allow collection staff to add their own alt text when creating poster/thumbnails from a video for display purposes.

**Comment by charumitraravi** [comment]
QA verified

**Comment by elynema** [comment]
Charu noted that although the image is skipped when traversing the page with a screen reader (MacOS voiceover and the Chrome plugin) as we intended, weird things do happen if you click on the thumbnail image while a screenreader is turned on.

When you click on the thumbnail image, the screen reader says ‘article.’ This is somewhat confusing for a user who is using a screenreader with a mouse (perhaps someone who is low vision but still uses a mouse?).
